defaults/main/logind.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
The use of the dictionary, in my opinion, was not appropriate, because when we need to edit just 1 parameter, we end up having to declare all the others. This is horrible, I noticed this a lot when I went to customize sysctl values. :'( I would have to edit a lot of things, but it would be good to change things little by little. If you know of a viable and elegant way to make this change, without requiring an extra task, please show an example.

Reply:
yeah, you're right @cleberb 
i'll see if i can find a better way of doing it, it's just makes the parameters easier to read in my opinion.

Reply:
i would like ansible to handle it all like `yq` does.

```sh
$ cat group_vars/all/test_vars.yml host_vars/localhost/mod_vars.yml
---
ipv4_sysctl_settings:
  net.ipv4.conf.all.accept_redirects: 0
  net.ipv4.conf.all.accept_source_route: 0
  net.ipv4.conf.all.log_martians: 3
---
ipv4_sysctl_settings:
  net.ipv4.conf.all.log_martians: 0
  net.ipv4.conf.default.log_martians: 0
  this.is.a.test: 1
```
```sh
$ yq m -x group_vars/all/test_vars.yml host_vars/localhost/mod_vars.yml
ipv4_sysctl_settings:
  net.ipv4.conf.all.accept_redirects: 0
  net.ipv4.conf.all.accept_source_route: 0
  net.ipv4.conf.all.log_martians: 0
  net.ipv4.conf.default.log_martians: 0
  this.is.a.test: 1
```

Compared to the `ansible` result:

```sh
$ ansible-playbook -c local -i 'localhost,' test.yml

PLAY [Various Ansible tests] *****************************************************************

TASK [Get sysctl settings] ******************************************************************
ok: [localhost] => {
    "msg": {
        "net.ipv4.conf.all.log_martians": 0,
        "net.ipv4.conf.default.log_martians": 0,
        "this.is.a.test": 1
    }
}

PLAY RECAP ******************************************************************************
localhost                  : ok=1    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0
```

Reply:
https://github.com/ansible/ansible/issues/84009

Reply:
@cleberb see https://github.com/ansible/ansible/issues/84009#issuecomment-2377940112